components/breadcrumb/index.css

```diff
@@ -18,35 +18,35 @@ governing permissions and limitations under the License.
   --spectrum-breadcrumbs-block-size-multiline: var(--spectrum-breadcrumbs-height-multiline);
 
   /* text regular */
-  --spectrum-breadcrumbs-line-height: var(--spectrum-line-height-small);
+  --spectrum-breadcrumbs-line-height: var(--spectrum-line-height-100);
   --spectrum-breadcrumbs-font-size: var(--spectrum-font-size-200);
   --spectrum-breadcrumbs-font-family: var(--spectrum-font-family-base); /* placeholder for --spectrum-font-family-default */
-  --spectrum-breadcrumbs-font-weight: var(--spectrum-font-weight-regular); /* placeholder for --spectrum-font-regular */
+  --spectrum-breadcrumbs-font-weight: var(--spectrum-regular-font-weight); /* placeholder for --spectrum-font-regular */
```

Comment:
We can remove the notes on each of these that say 

`/* placeholder for --spectrum-font-regular */`

These were to ensure the original intended token name was recorded in the file.